app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm.rb

```diff
@@ -33,6 +33,13 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::Vm < Man
   end
   supports :launch_html5_console
 
+  supports :native_console do
+    reason ||= _("VM Console not supported because VM is orphaned") if orphaned?
+    reason ||= _("VM Console not supported because VM is archived") if archived?
+    unsupported_reason_add(:native_console, reason) if reason
+  end
+  supports :launch_native_console
```

Comment:
I don't believe this feature is what you're adding here (aka vm remote console)
```suggestion
```

Reply:
Good catch. Dropping...